src/gui/screen_home.cpp

```diff
@@ -114,7 +114,13 @@ void screen_home_init(screen_t *screen) {
         }
     }
 
-    if (!marlin_update_vars(MARLIN_VAR_MSK(MARLIN_VAR_MEDIAINS))->media_inserted)
+    if (pw->is_starting)
```

Comment:
please wrap the if() block into { } ... especially usefull with the multi-line comment